core/src/main/resources/reference.conf

```diff
@@ -23,6 +23,10 @@ akka {
     #parallelism-max = 64
   }
 
+  priority-mailbox {
+    mailbox-type = "akka.dispatch.UnboundedControlAwareMailbox"
```

Comment:
👁

Reply:
This allows messages that extend the `ControlMessage` trait to be processed immediately. I'll add a comment here